code/__DEFINES/preferences.dm

```diff
@@ -3,12 +3,11 @@
 #define SOUND_MIDI			2
 #define SOUND_AMBIENCE		4
 #define SOUND_LOBBY			8
-#define SOUND_STREAMING		16
-#define SOUND_HEARTBEAT		32
-#define SOUND_BUZZ			64
-#define SOUND_INSTRUMENTS	128
+#define SOUND_HEARTBEAT		16
+#define SOUND_BUZZ			32
+#define SOUND_INSTRUMENTS	64
```

Comment:
Changing bit flags like this will screw up preferences. Have to leave 16 unused for now.